re_data_ui/tsconfig.json

```diff
@@ -19,14 +19,18 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "typeRoots": [ "./types", "./node_modules/@types"],
+    "typeRoots": [
+      "./types",
+      "./node_modules/@types"
+    ],
     "downlevelIteration": true,
   },
   "include": [
-    "src"
+    "src",
+    "types/react-table/index.d.ts"
```

Comment:
there are some issues with ts detecting the types from the `@types/react-table` package?

Reply:
removed(it was added by my editor)

Reply:
okay, don't forget to update the pr.

Reply:
done